admin/qtx_admin_options.php

```diff
@@ -91,3 +91,39 @@ function qtranxf_admin_load_config() {
 
     qtranxf_add_conf_filters();
 }
+
+/**
+ * Import legacy option by recopying its value if the new option doesn't already exist.
+ *
+ * @param string $old_name
+ * @param string $new_name
+ * @param bool|string $autoload as in update_option
+ *
+ * @return void
+ */
+function qtranxf_import_legacy_option( $old_name, $new_name, $autoload = null ) {
```

Comment:
Should this function return `update_option` result (and `true` for other cases)?

Reply:
I haven't found a case where I'd need it.